src/cdromfs.c

```diff
@@ -117,18 +119,23 @@ static struct fs_dirent * cdrom_dirent_lookup( struct fs_dirent *dir, const char
 	int data_length = cddir->length;
 
 	struct iso_9660_directory_entry *d = (struct iso_9660_directory_entry *) data;
+	char *upper_name = kmalloc(strlen(name));
```

Comment:
strdup?

Reply:
Should I create a strdup for `string.h`?  Since those are user exposed, should I wait until Jeffrey's #80 is merged?  I don't see a way to do this before user processes can malloc.

I might be misunderstanding.

Reply:
This is kernel code, not user code, so `strdup` should be calling the existing `kmalloc`